.github/workflows/ci-test.yml

```diff
@@ -32,9 +32,9 @@ jobs:
         key: test-${{ matrix.unified }}-${{ runner.os }}-gcc
         max-size: 1000M
 
-    - name: Build (Ubuntu Linux, GCC)
+    - name: Build (Ubuntu Linux, GCC, Sanitizers)
```

Comment:
Just a nit, maybe also rename the top-level test to include this name. Then the label immediately shows in the action panel.